source/includes/usage-examples/code-snippets/WatchCompanion.java

```diff
@@ -1,3 +1,10 @@
+/**
```

Comment:
Q: I could be wrong here but are two asterisks needed in the first line? Applies to all